.github/workflows/label_merge_conflicts.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Merge Conflict Detection'
+on:
+  push:
+    branches:
+      - master
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mschilde/auto-label-merge-conflicts@master
+        with:
+          CONFLICT_LABEL_NAME: 'Merge Conflict'
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This will be automatically made when the action is merged in, no configuration is needed.